--- origsrc/gmp-5.0.5/configure.in	2012-05-06 06:19:49.000000000 -0500
+++ src/gmp-5.0.5/configure.in	2012-09-12 04:50:33.186529500 -0500
@@ -2335,16 +2335,6 @@ if test "${with_tags+set}" != set; then
   fi
 fi
 
-# The dead hand of AC_REQUIRE makes AC_PROG_LIBTOOL expand and execute
-# AC_PROG_F77, even when F77 is not in the selected with_tags.  This is
-# probably harmless, but it's unsightly and bloats our configure, so pretend
-# AC_PROG_F77 has been expanded already.
-#
-# FIXME: Rumour has it libtool will one day provide a way for a configure.in
-# to say what it wants from among supported languages etc.
-#
-AC_PROVIDE([AC_PROG_F77])
-
 AC_PROG_LIBTOOL
 
 # Generate an error here if attempting to build both shared and static when
